t/venom/Makefile

```diff
@@ -216,6 +216,13 @@ inline_l2_and_radius_deb11:
 	SCENARIOS_TO_RUN=inline_l2_and_radius \
 	$(MAKE_TARGET)
 
+samba4_deb11:
+	make \
+	PF_VM_NAMES=pfdeb11$(DEV_ENV) \
+	INT_TEST_VM_NAMES="linux03 switch01 node01 node03 wireless01" \
+        SCENARIOS_TO_RUN=dot1x_eap_peap_samba4 \
```

Comment:
We need to be consistent between name of make target, scenario and what we will define in `.gitlab-ci.yml`

Reply:
done